### DISCUSSION TITLE
is both juicefs.grouping /juicefs.users need config in core-site.xml ?

### DISCUSSION THREAD

**Discussion by user-GitHub-user** [juicedata/juicefs#3011] [Q&A] [answered]
1.is both juicefs.grouping /juicefs.users need config in core-site.xml ? 
2.in juicefs.grouping GID is required or not ?

**Comment by tangyoupeng** [comment, marked as answer]
If different hosts for the same user or group to use have different uid and gid, then you should configure juicefs.groups and juicefs.users.
Refer to: https://github.com/juicedata/juicefs/blob/main/docs/zh_cn/deployment/hadoop_java_sdk.md#%E5%85%B6%E5%AE%83%E9%85%8D%E7%BD%AE
